src/map_io/map_elemental_packet.cc

```diff
@@ -49,6 +49,7 @@ void MapElementalPacket::pre_read(FileSystem& fs, Map* map) {
 			map->set_hint(s.get_string("hint", ""));
 			map->set_background(s.get_string("background", ""));
 			old_world_name_ = s.get_string("world", "");
+			map->set_background_theme(s.get_string("theme", old_world_name_.c_str()));
```

Comment:
`old_world_name_` still needs to be mapped, like in `EditorInteractive::translate_old_world_name` - maybe move that function to the `Map` class?